Bottom: bf70d66b180246030ab13fa7e96ebe3a23ac8734
Top:    d1294febdb2765b409eade713be8583f6162a4dd
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-22 18:09:40 +0800

Refresh of vhost-descriptor-batch

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index 79c6e7a..807828a 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -84,6 +84,12 @@ enum vhost_uaddr_type {
 	VHOST_NUM_ADDRS = 3,
 };
 
+struct vhost_vring_desc {
+	struct vring_desc descs[64];
+	int head;
+	int tail;
+};
+
 /* The virtqueue structure describes a queue attached to a device. */
 struct vhost_virtqueue {
 	struct vhost_dev *dev;
@@ -150,6 +156,8 @@ struct vhost_virtqueue {
 	bool user_be;
 #endif
 	u32 busyloop_timeout;
+
+	struct vhost_vring_desc descq;
 };
 
 struct vhost_msg_node {
